docs/analytics_spec.md

```diff
@@ -48,22 +95,14 @@ This board shows metrics that relate to the entire marketplace, such as:
 - started auctions sorted by lot
 - bids placed sorted by submitting wallet
 
-### Seller Board
-This board would show metrics relevant to the seller, to be displayed when a user views an auction that they listed:
-
-- auction views filtered by auctionId
-- deposits placed filtered by auctionId
-- bids placed filtered by auctionId
-- refunds filtered by auctionId
+### Integrating a Custom Board
+To replace the default board with your own custom Mixpanel board, only the .env needs to be changed.
 
-### Bidder Board
-This board would show metrics relevant to prospective bidders, to be displayed when a user views an auction that they haven't listed:
+Copy the board link and paste it into the `REACT_APP_MIXPANEL_BOARD_LINK`. The board under the Analytics tab should show your board after this change and restarting the server.
 
-- announced auctions filtered by lot
-- deposits placed filtered by auctionId
-- bids placed filtered by auctionId
+Note: This embedding will only work with a public Mixpanel board.
 
-__TODO__: Create Board Embedding Tutorial
+Once your project has been created, you'll be able to use your project token to connection your adapted hydra-auction to your mixpanel project, and should see our premade events begin to appear in your Mixpanel dashboard.
```

Comment:
Minor - should Hydra-Auction and Mixpanel be capitalized throughout the document for consistency?